page.codeberg.foreverxml.Random.json

```diff
@@ -0,0 +1,80 @@
+{
+    "app-id" : "page.codeberg.foreverxml.Random",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "randomgtk",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "libsass",
```

Comment:
```
]2;flatpak-builder: Cleanup libsassError: module: libsass: Can't find meson.build
```

Shouldn't it be autotools? Also why does it use a fork instead of upstream repo?

Reply:
I copied it from the internet (quite sadly) but I have fixed it.